src/partials/categories/CreateCategory.svelte

```
@@ -0,0 +1,103 @@
+</script>
+
+<button
+  type="button"
+  class="inline-flex items-center justify-center gap-x-2 rounded-lg border border-transparent text-sm focus:outline-none disabled:pointer-events-none disabled:opacity-50 {triggerBtnColor}"
+  aria-haspopup="dialog"
+  aria-expanded="false"
+  aria-controls="category-modal"
+  data-hs-overlay="#category-modal"
+  {onclick}>
+  {#if create}
+    {@html plusIcon}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)